src/app/login/login.component.ts

```diff
@@ -0,0 +1,31 @@
+import {Component, OnInit} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+
+@Component({
+  selector: 'app-login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+
+export class LoginComponent implements OnInit{
+
+  loginForm: FormGroup
+
+  ngOnInit() {
+    this.loginForm = new FormGroup({
```

Comment:
In angular 14+ where we have typed forms it is good to create `FormGroup` inline.

Reply:
done :)